SwiftWisdom/Core/StandardLibrary/Numbers/Double+Extensions.swift

```diff
@@ -2,13 +2,14 @@
 //  Double+Extensions.swift
 //  SwiftWisdom
 //
-//  Created by Stephen Wong on 5/1/16.
-//  Copyright © 2016 Intrepid. All rights reserved.
+//  Created by Litteral, Maximilian on 2/22/19.
+//  Copyright © 2019 Intrepid. All rights reserved.
 //
 
 import Foundation
 
 extension Double {
+    @available(*, unavailable, renamed: "ip_rounded(toDecimalPlaces:)")
     public func ip_round(toDecimalPlaces decimalPlaces: Int) -> Double {
         let decimalShiftBase10 = pow(10.0, Double(decimalPlaces))
```

Comment:
Since this code now cannot be called, it should be removed. But since the method return a value, it needs to call `fatalError` or something similar.